src/cli.cr

```diff
@@ -86,14 +93,15 @@ module Shards
         targets << arg
       end
     end
+    { targets, options }
+  end
 
+  def self.check_and_install_dependencies(path)
     begin
```

Comment:
This `begin` / `end` became redundant now.

Reply:
Could you elaborate on this? It appears to me that the check command can still raise an error that should be resolved by doing the install command.

Reply:
yes, I mean that you can use `rescue` without `begin ... end`:

```cr
def self.check_and_install_dependencies(path)
  # ...
rescue # ...
  # ...
end
```

Reply:
Great, thanks. I didn't know that was possible to do 👍